src/ParticlePatches.cpp

```diff
@@ -69,7 +69,7 @@ ParticlePatches::read()
         IOHandler->flush();
 
         using DT = Datatype;
-        if( DT::UINT64 != *dOpen.dtype )
+        if( DT::ULONG != *dOpen.dtype )
```

Comment:
Correct me if I'm wrong, but this does not strictly enforce the standard, `ULONG` may be 32 or 64.
```cpp
if( determineDatatype< uint64_t > != *dOpen.dtype )
```

Reply:
exactly, that's the leftover from my replace marathon and your suggestion is correct!